libs/agno/agno/models/meta/llama.py

```diff
@@ -153,13 +153,6 @@ def request_kwargs(self) -> Dict[str, Any]:
         if self._tools is not None and len(self._tools) > 0:
             request_params["tools"] = self._tools
 
-            # Fix optional parameters where the "type" is [<type>, null]
```

Comment:
wasn't this added when reasoning tools were not working with Llama, this does not correlate to it then?

Reply:
Yes this was added, but I removed the issue that this is "patching"